release_util/management/commands/show_unapplied_migrations.py

```diff
@@ -17,7 +17,9 @@ class Command(BaseCommand):
       - [app1, 0001_initial]
       - [app2, 0012_otherthing]
       - [app1, 0002_somthing]
-    initial_states: {app1: zero, app2: 0011_thisthing}
+    initial_states:
```

Comment:
The output of `MigrationSession.list` matches the pre-serialization output of this command because I just copypasta'd it into MigrationSession.